package.json

```diff
@@ -2,19 +2,22 @@
   "name": "solc",
   "version": "0.8.11",
   "description": "Solidity compiler",
-  "main": "index.js",
+  "main": "dist/index.js",
   "bin": {
-    "solcjs": "solc.js"
+    "solcjs": "dist/solc.js"
   },
   "scripts": {
-    "lint": "eslint .",
-    "lint:fix": "eslint --fix .",
-    "updateBinary": "node downloadCurrentVersion.js && node verifyVersion.js",
-    "prepublishOnly": "npm run updateBinary",
-    "pretest": "npm run lint",
-    "test": "tape ./test/index.js",
-    "coverage": "node ./node_modules/nyc/bin/nyc.js --reporter=lcov --reporter=text-summary ./node_modules/tape/bin/tape ./test/index.js",
-    "coveralls": "npm run coverage && node ./node_modules/coveralls/bin/coveralls.js <coverage/lcov.info"
+    "build": "tsc",
+    "postbuild": "node build/postbuild.js",
+    "lint": "eslint --ext .js,.ts .",
+    "lint:fix": "eslint --fix --ext .js,.ts .",
+    "updateBinary": "node build/clean.js && ts-node ./downloadCurrentVersion.ts && ts-node ./verifyVersion.ts",
+    "prepublishOnly": "npm run updateBinary npm && npm run build",
+    "copyTestFiles": "cp -r ./test/resources ./dist/test/",
```

Comment:
Hmm, there is also `test/smtCheckerTests`. That doesn't need to be copied?

Reply:
This copies test resources not tests, it wont make it into dist unless its a typescript file

Reply:
This has now been resolved.

Reply:
Since #589 was merged (created it yesterday), can you please rebase? It seems you've done the same here.

Reply:
done